### DISCUSSION TITLE
Dynamic Topic Model: Accessing c-tf-idf

### DISCUSSION THREAD

**Discussion by Spekkboom** [MaartenGr/BERTopic#1459] [Q&A] [answered]
Hi!

Is there any way to easily access the adjusted c-tf-idf that is calculated for each topic in each timestep when using the `topics_over_time()` function?

**Comment by MaartenGr** [comment, marked as answer]
That is currently not possible since with many timesteps the resulting c-TF-IDF matrices could potentially explode in size. Having said that, I think it should be straightforward to adjust `.topics_over_time` and return those. I would advise checking the issues, I remember there already was an issue with an example of this.

**Reply by Spekkboom** [reply]
Ah yes, that makes sense. Thank you, I'll adjust the function for my use case.
